tests/src/test/scala/ironrust/FullIntegrationTest.scala

```diff
@@ -446,17 +446,23 @@ class FullIntegrationTest extends DudeSuite with CancelAfterFailure {
       result.getChanged.getGroups should have length 0
     }
 
-    "grant to specified groups" in {
+    "grant to specified groups and INTERNAL/PII policy" in {
       val sdk = IronSdk.initialize(createDeviceContext)
       val data: Array[Byte] = List(1, 2, 3).map(_.toByte).toArray
       val maybeResult = Try(
-        sdk.documentEncrypt(data, DocumentEncryptOpts.create(null, null, true, Array(), Array(validGroupId)))
+        sdk.documentEncrypt(data, DocumentEncryptOpts.create(null, null, true, Array(), Array(validGroupId), new PolicyGrant(Category.validate("PII"), Sensitivity.validate("INTERNAL"), null, null)))
```

Comment:
Probably some thoughts for down the road, but maybe they will resonate with you... Would it be better to provide more `DocumentEncryptOps` constructors to reduce the use of null here?
For `PolicyGrant` would it be useful to provide something like `Category.None` for those not wanting to use null. Of course we can't actually stop a caller from passing null, but maybe we could give them some examples that aren't as foot-cannony.

Reply:
I think instead of constructors it might be nice to supply static methods with good names, but yes it would be good to make this less ugly. Having `Category.None` which returns null seems worse than explicit null here. It implies that it's a real value. Things like `Category.None.toString` would throw, which stinks. 

I really dislike that null is used to signify None, but at least we have nice Scala bindings.